frodo.js

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
SCRIPT-This function might benefit from using async/await instead of callbacks for better readability.